src/libraries/System.IO.Ports/tests/System.IO.Ports.Tests.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Linux;$(NetCoreAppCurrent)-OSX;$(NetCoreAppCurrent)-FreeBSD;$(NetFrameworkMinimum)</TargetFrameworks>
+    <EnableRegexGenerator>true</EnableRegexGenerator>
```

Comment:
Note the TargetFrameworks line above. As it contains things other than NetCoreAppCurrent, the changes in this project won't work.

Reply:
I know - I'd made a pass previously, but seem to have been constrained for time, and forgot I hadn't finished it.